clients-simple/src/main/java/uk/gov/gchq/palisade/clients/simpleclient/web/UrlDataClient.java

```diff
@@ -45,7 +45,7 @@ public DataClient build(final String serviceId) {
         Map<String, String> dataServices = clientConfiguration.getClient();
         return feignClientBuilder
                 .forType(DataClient.class, serviceId)
-                .url(dataServices.getOrDefault(serviceId, serviceId))
+                .url(dataServices.getOrDefault(serviceId, "http://" + serviceId))
```

Comment:
Very minor point, I think the the http:// is a constant and so I would have expected it to be a declared as such as the class variable.  

More of a question, but the separation of the http:// from the variable serviceId means that I cannot pass a serviceId that includes https:// as part of the URL.  Do we care?

Reply:
all of our service names so far are just "service-name" and there are no requirements for this to change. but it can be addressed going forwards